Formula/hivemind.rb

```diff
@@ -0,0 +1,22 @@
+class Hivemind < Formula
+  desc "The mind to rule processes of your development environment"
+  homepage "https://github.com/DarthSim/hivemind"
+  url "https://github.com/DarthSim/hivemind/archive/v1.0.tar.gz"
+  sha256 "7dde50f8f68214929f53c380ffc6311b39aad071a67bb6d94ebc3e641f78083a"
+
+  head "https://github.com/DarthSim/hivemind.git"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/DarthSim/hivemind/").install Dir["*"]
+    system "go", "build", "-o", "#{bin}/hivemind", "-v", "github.com/DarthSim/hivemind/"
+  end
+
+  test do
+    actual = pipe_output("#{bin}/hivemind")
+    expected = "\e[1;31mopen ./Procfile: no such file or directory\e[0m\n"
```

Comment:
Could you consider writing a `Procfile` out instead and running a simple command? Thanks!